src/modules/UI/scenes/Plugins/EdgeProvider.js

```diff
@@ -171,48 +174,28 @@ export class EdgeProvider extends Bridgeable {
       this.selectedCurrencyCode = currencyCode
       this.selectedChainCode = chainCode
 
-      let returnCurrencyCode
+      const unfixCode = unfixCurrencyCode(this._plugin.fixCurrencyCodes, pluginId, tokenId)
+      if (unfixCode != null) {
+        return unfixCode
+      }
 
-      if (allowedCurrencyCodes != null && allowedCurrencyCodes.length > 0 && allowedCurrencyCodes.every(code => typeof code === 'object')) {
+      if (allObjects) {
```

Comment:
allObjects is true if allowedCurrencyCodes is an empty array because .every() always returns true on empty arrays. Given the param name, and current usage (ie. Banxa), we should return a currency code when allowedCurrencyCodes is empty. I wrote this exact same Banxa breaking bug previously.

Reply:
I was aware  that empty arrays always return true, but did not realize Banxa passed an empty array. I thought they passed undefined. Now they actually pass an array with values. 

Ok I’ll make the update to check for empty array in case others do that